scripts/apps/workspace/content/directives/ContentProfileSchemaEditor.ts

```diff
@@ -126,10 +126,9 @@ export function ContentProfileSchemaEditor(vocabularies) {
 
             scope.getEditor3FormattingOptions = (fieldName) => {
                 const isCustomPlainTextField = typeof scope.fields[fieldName] === 'object'
-                    && typeof scope.fields[fieldName].field_options === 'object'
-                    && scope.fields[fieldName].field_options.single === true;
+                    && scope.fields[fieldName].field_type === 'text';
```

Comment:
What other `field_type`s there are?

Reply:
`related_content`, `media`, `date_field` etc.

Reply:
> related_content, media, date_field etc.

None of these can use editor3, does it make sense to check them?

Reply:
Not at all, we have to check it for `text` fields only.